app/screens/login/login.js

```diff
@@ -437,6 +437,7 @@ export default class Login extends PureComponent {
 const style = StyleSheet.create({
     container: {
         flex: 1,
+        backgroundColor: 'white',
```

Comment:
why are we setting the background color to white? shouldn't this screen follow the OS sync theme dark/light?